src/bundler.ts

```diff
@@ -0,0 +1,28 @@
+import { exec } from "child_process";
```

Comment:
@vinistock Is this what you were envisioning when you mentioned a `bundler.ts` file?

Reply:
(And assuming you like this pattern I can make a follow up PR moving all the other bundler methods in here)